cmd/kind/build/build.go

```diff
@@ -30,7 +30,7 @@ func NewCommand() *cobra.Command {
 		// TODO(bentheelder): more detailed usage
 		Use:   "build",
 		Short: "Build one of [base-image, node-image]",
-		Long:  "Build one of [base-image, node-image]",
+		Long:  "Build the base node image (base-image) or the node image (node-image)",
```

Comment:
what does this look like at the CLI when printed? 
the short usage listing the valid options seems really handy, though obviously it would be great to also explain those options :-)

Reply:
usually OK to omit long if short fits on a single line.

Reply:
![kind-pr103](https://user-images.githubusercontent.com/7906467/48450730-fbb18700-e76c-11e8-8a87-83eb0fabd9a1.png)
 What do you think?

Reply:
Looks good to me, thanks!